site2/docs/cookbooks-partitioned.md

```diff
@@ -65,7 +65,7 @@ String topic = "persistent://my-tenant/my-cluster-my-namespace/my-topic";
 
 PulsarClient client = PulsarClient.create(pulsarBrokerRootUrl);
 ProducerConfiguration config = new ProducerConfiguration();
-config.setMessageRouter(AlwaysTenRouter);
+config.setMessageRouter(new AlwaysTenRouter());
```

Comment:
Ouch, this should have already been changed to the new preferred API : 

`client.newProducer().messageRouter(...).create()`

Can you fix it in this PR?

Reply:
Yep, will be addressing this as well ;)